atlascope/core/models/dataset.py

```diff
@@ -115,6 +118,18 @@ def create(self, validated_data):
     )
 
 
+class DatasetSubImageSerializer(serializers.Serializer):
+
+    original_dataset_id = serializers.UUIDField(required=True)
+    x0 = serializers.IntegerField(required=True)
+    y0 = serializers.IntegerField(required=True)
+    x1 = serializers.IntegerField(required=True)
+    y1 = serializers.IntegerField(required=True)
+
+    def validate_original_dataset_id(self, value):
+        return str(value)
```

Comment:
I think this should validate that `original_dataset_id` actually refers to a dataset.